include/odp/api/spec/event.h

```diff
@@ -209,6 +209,40 @@ void odp_event_free_multi(const odp_event_t event[], int num);
  */
 void odp_event_free_sp(const odp_event_t event[], int num);
 
+/**
+ * Event flow id value
+ *
+ * Returns the flow id value set in the event.
+ * Flow id value set in the event is used by scheduler to distribute the event
+ * across different flows. For packets received through packet input initial
```

Comment:
This is not accurate specification of flow id usage:
"Flow id value set in the event is used by scheduler to distribute the event across different flows."

Scheduler uses queue + flow id to maintain sync context. So may be replace with this:
"Usage of flow id enables scheduler to maintain multiple synchronization contexts per single queue. For example, when multiple flows are assigned to an atomic queue, events of a single flow (events from the same queue with the same flow id value) are guaranteed to be processed by only single thread at a time."